cmd/kube-scheduler/app/server.go

```diff
@@ -138,7 +138,7 @@ func runCommand(cmd *cobra.Command, opts *options.Options, registryOptions ...Op
 // Run executes the scheduler based on the given configuration. It only returns on error or when context is done.
 func Run(ctx context.Context, cc *schedulerserverconfig.CompletedConfig, sched *scheduler.Scheduler) error {
 	// To help debugging, immediately log version
-	klog.V(1).InfoS("Starting Kubernetes Scheduler version", "version", version.Get())
```

Comment:
what if you just remove `V(1).` and keep the structured logging? how is that going to look like?

Reply:
> what if you just remove `V(1).` and keep the structured logging? how is that going to look like?

"Starting Kubernetes Scheduler version" version=1.22.0

Reply:
thanks, then lets just remove "version" from the sentence and keep the rest as is. Likely useful to use structured logging, makes it easier to parse if needs be.

Reply:
@ahg-g i have modify it,can you help to mergeit, thanks.